rules/php70/tests/Rector/FuncCall/NonVariableToVariableOnFunctionCallRector/Fixture/keep_curly_braces.php.inc

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Rector\Php70\Tests\Rector\FuncCall\NonVariableToVariableOnFunctionCallRector\Fixture;
+
+final class ModelSmokeTest
```

Comment:
The class should be named with same words as in file.

Hm... I know it's not much obvious.
 Could you make a CI rule for that?

Reply:
I will make CI rule in new task.